WalletWasabi.Fluent/Views/Shell/NetworkBadgeView.axaml

```diff
@@ -0,0 +1,27 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:viewModels="clr-namespace:WalletWasabi.Fluent.ViewModels"
+             x:DataType="viewModels:MainViewModel"
+             x:Class="WalletWasabi.Fluent.Views.Shell.NetworkBadgeView"
+             x:CompileBindings="True">
+
+  <UserControl.Styles>
+    <Style Selector="Border.NetworkBadge">
+      <Setter Property="CornerRadius" Value="3" />
+      <Setter Property="VerticalAlignment" Value="Center" />
+      <Setter Property="HorizontalAlignment" Value="Right" />
+      <Setter Property="BorderBrush" Value="Gold" />
+      <Setter Property="BorderThickness" Value="2" />
+    </Style>
+    <Style Selector="Border.NetworkBadge TextBlock">
+      <Setter Property="Margin" Value="5" />
+      <Setter Property="FontWeight" Value="Bold" />
+      <Setter Property="Foreground" Value="Gold" />
```

Comment:
```suggestion
      <Setter Property="FontWeight" Value="Thin" />
      <Setter Property="Foreground" Value="White" />
```

Reply:
Let's not use light font weight here, it doesnt look good with the font right now